src/test/kotlin/sirius/db/jdbc/SmartQuerySortingTest.kt

```diff
@@ -157,10 +145,26 @@ class SmartQuerySortingSpec extends BaseSpecification {
      * @param index the current index of the for loop
      * @return int based on the index
      */
-    private int generateMixedInt(int index) {
+    private fun generateMixedInt(index: Int): Int {
         if (index % 2 == 0) {
-            return index+2;
+            return index + 2
         }
         return index
     }
+
+    companion object {
+        @Part
+        private lateinit var oma: OMA
+
+        private const val TWO_BLOCKS_LENGTH = 2000
+        private const val ONE_AND_HALF_BLOCK_LENGTH = 1500
+        private const val ONE_BLOCK_LENGTH = 1000
+        private const val FIRST_COL = 0
+        private const val SECOND_COL = 1
+        private const val NO_COL = -1
+
+        fun cleanup() {
+            oma.select(SmartQueryTestSortingEntity::class.java).delete()
+        }
```

Comment:
**Required:** I think this needs to be annotated with AfterAll so it is actually executed by jUnit